src/test/scala/com/fulcrumgenomics/umi/ReviewConsensusVariantsTest.scala

```diff
@@ -239,4 +247,26 @@ class ReviewConsensusVariantsTest extends UnitSpec {
     // The metrics don't currently contain reads with spanning deletions, so D/1 is present above and absent below
     metrics.toIndexedSeq.map(_.consensus_read) should contain theSameElementsAs Seq("A/1", "B/1", "E/1", "F/1", "H/1", "H/2")
   }
+
+  it should "fail if a FASTA sequence dictionary (.dict) is not present" in {
+    val outBase   = makeTempFile("review_consensus.", ".out")
+    val intervals = makeTempFile("empty.", ".vcf")
+    val ref       = buildRefFasta(withDict=false)
+
+    val vcfBuilder = VariantContextSetBuilder("s1")
+    vcfBuilder.header.setSequenceDictionary(header.getSequenceDictionary)
+    vcfBuilder.write(intervals)
+    an[ValidationException] should be thrownBy new ReviewConsensusVariants(input=intervals, consensusBam=consensusBam, groupedBam=rawBam, ref=ref, output=outBase)
```

Comment:
Scalatest has syntax to make the DSL appear grammatically correct FYI:

```suggestion
    a[ValidationException] should be thrownBy new ReviewConsensusVariants(input=intervals, consensusBam=consensusBam, groupedBam=rawBam, ref=ref, output=outBase)
```